kolibri/auth/permissions/auth.py

```diff
@@ -48,20 +48,20 @@ def user_can_delete_object(self, user, obj):
             return super(CollectionSpecificRoleBasedPermissions, self).user_can_update_object(user, obj.parent)
 
 
-class AnonUserCanReadFacilitiesThatAllowSignUps(DenyAll):
+class AnonUserCanReadFacilities(DenyAll):
```

Comment:
Merits discussion if we are ok with this approach.

Reply:
Permission-wise, I think it's OK, but see other comments about public vs internal API endpoints.